patches/ungoogled-chromium/macos/fix-gn-safe_browsing.patch

```diff
@@ -116,20 +118,18 @@
    }
 --- a/chrome/browser/ui/webui/settings/change_password_handler.cc
```

Comment:
Nitpick: Adding C++ code changes for safe browsing conflicts with this patch's original intent. Two options:

* Move the current C++ changes to `ungoogled-chromium` (unless the changes do not benefit any other platform or break other platforms)
* Rename the patch to `fix-disabling-safebrowsing.patch` (and fix the description accordingly)

Thoughts?

Reply:
When building for the portablelinux target, the changes were not necessary, so I don't think that they should necessarily be moved to `ungoogled-chromium`. Either renaming the patch or moving the changes out to another macOS specific patch would be how I would prefer to handle this.

Reply:
I see. I think we should just rename the patch, because splitting the patch based only on the code modified is not a strong enough reason.

Reply:
Fair enough. I'll rename the patch and push shortly.